scripts/gh-action-test/trigger_test.sh

```diff
@@ -31,7 +31,7 @@ echo "sending coins to different keys"
   --keyring-backend=test \
   --chain-id="testing" \
   --gas=auto \
-  --gas-prices="1905nhash" \
+  --gas-prices="1nhash" \
   --gas-adjustment=1.5 \
```

Comment:
And this file with the `--gas-adjustment` flags.

Reply:
:heavy_check_mark: